.github/workflows/scripts/lint-changed.sh

```diff
@@ -52,7 +55,16 @@ done <<< "$raw_changed_files"
 for file in "${changed_files[@]}"; do
     for ext in "${ESLINT_SUPPORTED_EXT[@]}"; do
         if [[ $file == *$ext ]]; then
-            supported_files+=("$file")
+            pattern_match=0
```

Comment:
Shall we use a boolean here.

```suggestion
            is_supported=false
```

Reply:
https://github.com/wso2/identity-apps/pull/3981/commits/3dbd94ccd8112770bbc586a136fae02267c875fe